src/styles.scss

```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
```

Comment:
Do we really need this? If the theme is defined in the application, won't these components use it?

Reply:
this is needed for the application, not for the components